sos/report/plugins/__init__.py

```diff
@@ -1442,7 +1446,7 @@
         self.archive.add_node(path, mode, os.makedev(dev_maj, dev_min))
 
     # Methods for copying files and shelling out
-    def _do_copy_path(self, srcpath, dest=None):
+    def _do_copy_path(self, srcpath, dest=None, force=False):
```

Comment:
I guess it is due to https://github.com/sosreport/sos/pull/3245/files#diff-de713027a3719cdb2b535a39fe93171a78d1ec8c7ee1b3ae91d4ffeb784c9bf2L1455 :+1: 

        if self._is_forbidden_path(srcpath):
            self._log_debug("skipping forbidden path '%s'" % srcpath)
            return ''

with the only non-empty return value. Since we dont use return value, why the '' here (from past, no change in current PR)?

Reply:
Not sure, let's take care of that one way or another in another PR.